ClientApp/src/modules/authorization/views/CreateUserView/CreateUserView.tsx

```diff
@@ -69,6 +105,7 @@ const CreateUserView: React.FC = ({children}) => {
                 </Center>
                 {showErrorMatching()}
                 {showErrorNotNull()}
+                {showTelphoneError()}
```

Comment:
Inline conditional rendering is better; less defined functions in component

Reply:
are you saying to remove function and replace it with component?

Reply:
Yeah. I that way component logic will not be mixed with html components